httpx/client.py

```diff
@@ -153,7 +153,7 @@ def merge_headers(
             return merged_headers
         return headers
 
-    async def send(
+    async def _send(
```

Comment:
I'm not sure I understand the reasoning of this change, can we not keep the `send()` methods the same as they were previously?

Reply:
we need some internal method for sending request
also we need different method with different signature 
`send() -> Response` and `send() -> AsyncResponse`

Reply:
You're right! I take back this comment, although this method might need a better name?